content/docs/reference/pytorchjob/v1beta1/pytorch.md

```diff
@@ -0,0 +1,226 @@
++++
+title = "PyTorchJob"
+description = "Reference documentation for PyTorchJob"
+weight = 100
++++
+<p>Packages:</p>
+<ul>
+<li>
+<a href="#kubeflow.org">kubeflow.org</a>
+</li>
+</ul>
+<h2 id="kubeflow.org">kubeflow.org</h2>
+<p>
+<p>Package v1beta1 is the v1beta1 version of the API.</p>
+</p>
+Resource Types:
+<ul><li>
+<a href="#PyTorchJob">PyTorchJob</a>
+</li></ul>
+<h3 id="PyTorchJob">PyTorchJob
+</h3>
+<p>
+<p>PyTorchJob represents the configuration of PyTorchJob</p>
+</p>
+<table>
+<thead>
+<tr>
+<th>Field</th>
+<th>Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>
+<code>apiVersion</code></br>
+string</td>
+<td>
+<code>
+kubeflow.org/v1beta1
+</code>
+</td>
+</tr>
+<tr>
+<td>
+<code>kind</code></br>
+string
+</td>
+<td><code>PyTorchJob</code></td>
+</tr>
+<tr>
+<td>
+<code>metadata</code></br>
+<em>
+<a href="https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.13/#objectmeta-v1-meta">
+Kubernetes meta/v1.ObjectMeta
+</a>
+</em>
+</td>
+<td>
+<p>Standard object&rsquo;s metadata.</p>
+Refer to the Kubernetes API documentation for the fields of the
+<code>metadata</code> field.
+</td>
+</tr>
+<tr>
+<td>
+<code>spec</code></br>
+<em>
+<a href="#PyTorchJobSpec">
+PyTorchJobSpec
+</a>
+</em>
+</td>
+<td>
+<p>Specification of the desired behavior of the PyTorchJob.</p>
+<br/>
+<br/>
+<table>
+<tr>
+<td>
+<code>cleanPodPolicy</code></br>
+<em>
+<a href="/docs/reference/tfjob/v1beta1/common/#CleanPodPolicy">
```

Comment:
It seems odd to link to a tfjob doc here. Actually, doing a search shows that there are a number of references to tfjob. Is this a copy/paste error?

Reply:
This is because the "common" API types are currently part of the tf-operator repository. We have a proposal to move these to a standalone repo: https://github.com/kubeflow/tf-operator/issues/960.